### PR TITLE
BaSP-TG: Projects-Unit Test-DELETE PUT

### PR DESCRIPTION
**Pull request by Julian25**
Work done on this brach:

-  Projects.test.js file  added to test folder

- Data base Projects mocked

- Projects DELETE and PUT endpoints tested

- Projcts.js file  added to seed folder

### CODE REVIEW

File: src/tests/projects.test.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  });
+
+  test('should NOT update a Project when body is empty', async () => {
+    const response = await request(app).put(`/projects/${projectId}`).send();
+    expect(response.status).toBe(400);
+  });
+
+  test('Response should return a 400 status, error true if at least one required field is empty', async () => {
+    const response = await request(app).put(`/projects/${projectId}`).send(mockRequestBodyIncomplete);
+
```
Comment (lautaro23) [inline src/tests/projects.test.js:117]: Borrar espacio de mas  en linea 117.

File: src/tests/projects.test.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    const response = await request(app).get('/projects/localhost:3000/projects/6283baefcd44998f831522bb3333333333').send();
+    expect(response.status).toBe(404);
+    expect(response.error).toBeTruthy();
+  });
+});
+
+describe('Test PUT /projects', () => {
+  test('Response should return a 200 status, error false, and message has been successfully updated', async () => {
+    const response = await request(app).put(`/projects/${projectId}`).send(mockRequestBody);
+
```
Comment (lautaro23) [inline src/tests/projects.test.js:104]: Borrar espacio de mas  en linea 104.

File: src/tests/projects.test.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+      employees: [
+        {
+
+          role: 'DEV',
+          rate: '10',
+          id: '628578f0b38934591452aa2e',
+        },
+      ],
+    });
+
```
Comment (lautaro23) [inline src/tests/projects.test.js:137]: Borrar espacio de mas  en linea 137.

File: src/tests/projects.test.js [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+  });
+
+  test('Response should return a 400 status, error true if at least one required field is empty', async () => {
+    const response = await request(app).put(`/projects/${projectId}`).send(mockRequestBodyIncomplete);
+
+    expect(response.status).toBe(400);
+  });
+
+  test('Response should return a 404 status, error true, and message Project not found if the ID is inexistent', async () => {
+    const response = await request(app).put('/projects/628578f0b38934591452aa2e').send({
```
Comment (lautaro23) [inline src/tests/projects.test.js:122]: Dentro del .send utiliza la constante "mockRequestBodyIncomplete"

**Review by martumoser — APPROVED** [review]
PR APPROVE. GOOD JOB!

**Pull request merged**